root/etc/s6-overlay/s6-rc.d/init-code-server/run

```diff
@@ -26,9 +26,14 @@ if [[ ! -f /config/.profile ]]; then
     cp /root/.profile /config/.profile
 fi
 
-# fix permissions (ignore contents of /config/workspace)
-find /config -path /config/workspace -prune -o -exec chown abc:abc {} +
-chown abc:abc /config/workspace
+# fix permissions (ignore contents of workspace and cache folders)
+find /config \
+    -path "/config/workspace" -prune -o \
+    -path "/config/.npm" -prune -o \
+    -path "/config/.rustup" -prune -o \
+    -path "/config/.cargo" -prune -o \
```

Comment:
Could we just ignore any dot directories? Avoid having to change it for future exceptions? Or are there some we need to include?

Reply:
some of code server files are in dot directories like `.config` and `.cache`
But you're right, there could be a lot more dot directories for other dev languages and frameworks.
Perhaps we should do a plex treatment and only perform chown if the top folder owner does not match the PUID?

Reply:
Might be a sensible option unless we have a solid list of needed dot dirs we can include and exclude everything else. We can still chmod the .ssh keys and just ignore everything else.